hack/bazel-build.sh

```diff
@@ -26,6 +26,8 @@ rm -rf ${CMD_OUT_DIR}
 mkdir -p ${CMD_OUT_DIR}/virtctl
 mkdir -p ${CMD_OUT_DIR}/dump
 
+check_update_trust
```

Comment:
I am not sure we want to run this every time. It is also possible to run the scripts directly without docker.

Can we instead call it in hack/dockerized via `exec` at the right moment?

Reply:
And there also only if a custom ca is provided.

Reply:
There is already a check for this to only run if custom CA is provided. I will look though to see how to move this to an exec.